_articles/using-retries.md

```diff
@@ -1,193 +0,0 @@
-# Practical Circuit Breaking and Retries
```

Comment:
what's happening here?

Reply:
on master, we already have this guide, which was a mistaken addition in a PR that didn't link out anywhere. this PR turns this into an actual guide, and links it so that folks can read it